controls/microsoft-365-foundations-1.1.3.rb

```diff
@@ -0,0 +1,44 @@
+control 'microsoft-365-foundations-1.1.3' do
+    title 'Ensure that between two and four global admins are designated'
+    desc 'More than one global administrator should be designated so a single admin can be monitored and to provide redundancy should a single admin leave an organization. Additionally, there should be no more than four global admins set for any tenant. Ideally global administrators will have no licenses assigned to them.'
+
+    desc 'check'
+    'Ensure that between two and four global admins are designated:
```

Comment:
not sure if we do multiline strings like this with quotes or the %q syntax - talk with @wdower

Reply:
Yes we do, its standard ruby syntax